planning/behavior_velocity_planner/src/scene_module/virtual_traffic_light/scene.cpp

```diff
@@ -413,6 +424,15 @@ bool VirtualTrafficLightModule::modifyPathVelocity(
   // After stop_line
   state_ = State::PASSING;
 
+  // Check timeout after stop_line if the option is on
+  if (
+    planner_param_.check_timeout_after_stop_line && isStateTimeout(*virtual_traffic_light_state)) {
```

Comment:
The formatting seems strange... I'll check this.
Probably it's due to the version of clang-format.

The original one:
![image](https://user-images.githubusercontent.com/31987104/151501661-469c61be-638e-4fcd-9adf-aa08b9f7108b.png)

Reply:
In tier4/autoware.iv#2413, `uncrustify` was used.